src/js/containers/search/filters/psc/PSCSearchContainer.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+  * PSCSearchContainer.jsx
+  * Created by Emily Gullo 07/10/2017
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as searchFilterActions from 'redux/actions/search/searchFilterActions';
+
+import PSCSearch from 'components/search/filters/psc/PSCSearch';
+
+const propTypes = {
+    updateSelectedPSC: React.PropTypes.func
```

Comment:
Please use the new [PropTypes](https://github.com/facebook/prop-types#usage) module.